packages/@okta/vuepress-site/docs/reference/api/iam-roles/index.md

```diff
@@ -0,0 +1,3027 @@
+curl -v -X PUT \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/groups/00gsr2IepS8YhHRFf0g3/roles/JBCUYUC7IRCVGS27IFCE2SKO/targets/groups/00gsrhsUaRoUib0XQ0g3"
+```
+
+###### Response example
+
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+#### Remove a group target from a group administrator role
+
+##### Remove a group target from a group administrator role given to a user
+
+
+<ApiOperation method="delete" url="/api/v1/users/${userId}/roles/${roleId}/targets/groups/${groupId}" />
+
+Removes a Group target from a `USER_ADMIN` or `HELP_DESK_ADMIN` Role assigned to a User
+
+> **Note:** Don't remove the last Group target from a role assignment, as this causes an exception. If you need a role assignment that applies to all Groups, the API consumer should delete the `USER_ADMIN` role assignment and recreate it.
+
+###### Request parameters
+
+
+| Parameter   | Description                                | Param Type   | DataType   | Required |
+| :---------- | :----------------------------------------- | :----------- | :--------- | :------- |
+| groupId     | `id` of Group target for role assignment   | URL          | String     | TRUE     |
+| roleId      | `id` of a Role                             | URL          | String     | TRUE     |
+| userId      | `id` of a User                             | URL          | String     | TRUE     |
+
+###### Response parameters
+
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+###### Request example
+
+
+```bash
+curl -v -X DELETE \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/users/00u6fud33CXDPBXULRNG/roles/KVJUKUS7IFCE2SKO/targets/groups/00garkxjAHDYPFcsP0g4"
+```
+
+###### Response example
+
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+##### Remove a group target from a group administrator role given to a group
+
+<ApiOperation method="delete" url="/api/v1/groups/${groupId}/roles/${roleId}/targets/groups/${targetGroupId}" />
+
+Removes a Group target from a `USER_ADMIN` or `HELP_DESK_ADMIN` Role assigned to a Group
+
+> **Note:** Don't remove the last Group target from a role assignment, as this causes an exception. If you need a role assignment that applies to all Groups, the API consumer should delete the `USER_ADMIN` role assignment and recreate it.
+
+###### Request parameters
+
+
+| Parameter       | Description                                | Param Type   | DataType   | Required |
+| :-------------- | :----------------------------------------- | :----------- | :--------- | :------- |
+| groupId         | `id` of an admin Group                     | URL          | String     | TRUE     |
+| roleId          | `id` of a Role                             | URL          | String     | TRUE     |
+| targetGroupId   | `id` of Group target for role assignment   | URL          | String     | TRUE     |
```

Comment:
role assignment > Role assignment

General comment: Change `id` to just ID in the description.